WeakAurasOptions/ForAllIndentsAndPurposes.lua

```diff
@@ -974,16 +974,21 @@ if not IndentationLib.revision or revision > IndentationLib.revision then
 
     -- returns the padded code, and true if modified, false if unmodified
     local linebreak = stringbyte("\n")
+    local space = stringbyte(" ")
     function lib.padWithLinebreaks(code)
-        local len = stringlen(code)
-        if stringbyte(code, len) == linebreak then
-            if stringbyte(code, len - 1) == linebreak then
-                return code, false
+        local len = stringlen(code) + 1
```

Comment:
I am nitpicking, but this number doesn’t really represent the length of the string anymore.